lib/nerves_ssh.ex

```diff
@@ -18,6 +18,14 @@ defmodule NervesSSH do
 
   require Logger
 
+  @type opt ::
+          {:auto_start, boolean()}
+          | {:authorized_keys, [String.t()]}
```

Comment:
```suggestion
          {:authorized_keys, [String.t()]}
```
This went away, right?

Reply:
🤦 Derp. Yes, fixed